dlib/dnn/loss_abstract.h

```diff
@@ -2020,16 +2020,15 @@ namespace dlib
             Also, the loss value returned corresponds to the L2 loss.
         !*/
 
-        void adjust_nms
-        (
+        void adjust_nms (
             double iou_thresh,
             double percent_covered_thresh = 1,
             bool classwise = true
         );
         /*!
             ensures
-                - #get_options().overlaps_nms == test_box_overlap(iou_thresh, percent_covered_thresh)
-                - #get_options().classwise_nms == classwise
+                - #get_options().overlaps_nms = test_box_overlap(iou_thresh, percent_covered_thresh)
+                - #get_options().classwise_nms = classwise
```

Comment:
The convention in the dlib docs is to use == here, since the docs are saying "this expression will be true" rather than "this code will be executed".

Reply:
Right, reverted, thank you